Python/libraries/recognizers-text/recognizers_text/Matcher/AaNode.py

```diff
@@ -42,7 +42,9 @@ def fail(self, fail):
         self.__fail = fail
 
     def get_enumerator(self):
-        pass
+        child: Node
+        child.getEnumerator()
+        return self.__children.values.select(child)
```

Comment:
This should be done with a `map`
`map( lambda child: AaNode(child), children.values).getEnumerator`